packages/api/package.json

```diff
@@ -41,7 +42,10 @@
         "express-status-monitor": "1.3.4",
         "mocha": "10.2.0",
         "nodemon": "3.0.1",
-        "prettier": "3.0.0"
+        "prettier": "3.0.0",
+        "ts-node": "10.9.1",
+        "ts-patch": "3.0.2",
+        "typescript": "5.2.2"
```

Comment:
hmm, I think you don't need typescript here since you have it on dependencies

Reply:
you are right!